windows/INSTALL.md

```diff
@@ -73,10 +73,11 @@ For cygwin installation, see the instructions in `../INSTALL`.
    If you get this error: `Assertion failed: hunk, file ../patch-2.5.9-src/patch.c, line 354`
    it is because the `patch.c` file should have Windows line endings (CRLF) rather than Unix ones (LF).
    
-There are two options to use for BLAS (linear algebra): MLK and OpenBLAS. MLK is made by Intel and is optimised
-for their processors. Unfortunately it isn't free. OpenBLAS is free alternative with similar performance.
+There are two options to use for BLAS (linear algebra): [Intel® MKL](https://software.intel.com/en-us/intel-mkl) and OpenBLAS. [Intel® MKL](https://software.intel.com/en-us/intel-mkl) is made by Intel and is optimised
+for their processors. It isn't free, but you can get [Community Licensing for Intel® Performance Libraries
+](https://software.intel.com/sites/campaigns/nest/) or as part of Intel product suite if you [qualify as students, educators, academic researchers, and open source contributors](https://software.intel.com/en-us/qualify-for-free-software). OpenBLAS is free alternative with similar performance.
```

Comment:
I'm using Intel's Community License and they give Linux, OS X, and Windows variants.

These are the complete software provided by the Community License:

1. Intel® Integrated Performance Primitives (Intel® IPP) (1209 MB)
2. Intel® Data Analytics Acceleration Library(Intel® DAAL) (585 MB)
3. Intel® Math Kernel Library(Intel® MKL) (940 MB)
4. Intel® Threading Building Blocks(Intel® TBB) (137 MB)